book_service.py

```diff
@@ -1,20 +1,28 @@
 from typing import List
 import sqlite3
+from student import Student
 
+# def makeDB():
 BOOKS = 10
 
 conn = sqlite3.connect('books.db')
-
 cursor = conn.cursor()
 
-# cursor.execute(""" CREATE TABLE books (
-#             book_id integer PRIMARY KEY,
-#             owner text
-#         ) """)
+cursor.execute(""" CREATE TABLE IF NOT EXISTS books (
+            book_id integer PRIMARY KEY,
+            owner text
+        ) """)
+
+cursor.execute(" SELECT * FROM books ")
+table_size = len(cursor.fetchall())
 
-# for x in range(1, BOOKS + 1):
-#     cursor.execute("INSERT INTO books VALUES (?, ?)", (x, ''))
-#     conn.commit()
+# checks if the table is empty, if so generate some empty objects
+
+# print(table_size)
+if table_size == 0:
+    for x in range(1, BOOKS + 1):
+        cursor.execute("INSERT INTO books VALUES (?, ?)", (x, ''))
+        conn.commit()
```

Comment:
👍Good idea here inserting dummy values into the table. 

One pushback here is that you have restricted the number of books to be inserted to only 10. What if I want to add more books. How do I do that?

_I know this is just a dummy project but we want to think about that😎_